settings.gradle.kts

```diff
@@ -6,7 +6,15 @@ rootProject.name = "testng-root"
 
 plugins {
     `gradle-enterprise`
-    id("de.fayard.refreshVersions") version "0.10.0"
+    id("de.fayard.refreshVersions") version "0.30.1"
+////                            # available:"0.10.1"
```

Comment:
Remove old commented versions

Reply:
Thanks @juherr - I didnt know that. Fixed